### ISSUE TITLE
Compilation error

### ISSUE THREAD

**Issue by bduman** [AlexeyAB/darknet#816]
`nvcc -gencode arch=compute_37,code=sm_37  -DOPENCV `pkg-config --cflags opencv` -DGPU -I/usr/local/cuda/include/ -DCUDNN --compiler-options "-Wall -Wfatal-errors -Ofast -DOPENCV -DGPU -DCUDNN" -c ./src/network_kernels.cu -o obj/network_kernels.o
nvcc -gencode arch=compute_37,code=sm_37  -DOPENCV `pkg-config --cflags opencv` -DGPU -I/usr/local/cuda/include/ -DCUDNN --compiler-options "-Wall -Wfatal-errors -Ofast -DOPENCV -DGPU -DCUDNN" -c ./src/avgpool_layer_kernels.cu -o obj/avgpool_layer_kernels.o
gcc  -DOPENCV `pkg-config --cflags opencv` -DGPU -I/usr/local/cuda/include/ -DCUDNN -Wall -Wfatal-errors -Ofast -DOPENCV -DGPU -DCUDNN obj/gemm.o obj/utils.o obj/cuda.o obj/convolutional_layer.o obj/list.o obj/image.o obj/activations.o obj/im2col.o obj/col2im.o obj/blas.o obj/crop_layer.o obj/dropout_layer.o obj/maxpool_layer.o obj/softmax_layer.o obj/data.o obj/matrix.o obj/network.o obj/connected_layer.o obj/cost_layer.o obj/parser.o obj/option_list.o obj/darknet.o obj/detection_layer.o obj/captcha.o obj/route_layer.o obj/writing.o obj/box.o obj/nightmare.o obj/normalization_layer.o obj/avgpool_layer.o obj/coco.o obj/dice.o obj/yolo.o obj/detector.o obj/layer.o obj/compare.o obj/classifier.o obj/local_layer.o obj/swag.o obj/shortcut_layer.o obj/activation_layer.o obj/rnn_layer.o obj/gru_layer.o obj/rnn.o obj/rnn_vid.o obj/crnn_layer.o obj/demo.o obj/tag.o obj/cifar.o obj/go.o obj/batchnorm_layer.o obj/art.o obj/region_layer.o obj/reorg_layer.o obj/super.o obj/voxel.o obj/tree.o obj/convolutional_kernels.o obj/activation_kernels.o obj/im2col_kernels.o obj/col2im_kernels.o obj/blas_kernels.o obj/crop_layer_kernels.o obj/dropout_layer_kernels.o obj/maxpool_layer_kernels.o obj/network_kernels.o obj/avgpool_layer_kernels.o -o darknet -lm -pthread `pkg-config --libs opencv` -L/usr/local/cuda/lib64 -lcuda -lcudart -lcublas -lcurand -lcudnn -lstdc++
obj/image.o: In function `get_image_from_stream_resize':
image.c:(.text+0x2b0a4): undefined reference to `get_webcam_frame'
image.c:(.text+0x2b0d4): undefined reference to `get_webcam_frame'
obj/data.o: In function `load_data_detection':
data.c:(.text+0x34bf): undefined reference to `image_data_augmentation'
obj/network.o: In function `resize_network':
network.c:(.text+0x29b6): undefined reference to `resize_yolo_layer'
network.c:(.text+0x2a08): undefined reference to `resize_upsample_layer'
obj/network.o: In function `num_detections':
network.c:(.text+0x3722): undefined reference to `yolo_num_detections'
obj/network.o: In function `fill_network_boxes':
network.c:(.text+0x3d88): undefined reference to `get_yolo_detections'
obj/network.o: In function `custom_get_region_detections':
network.c:(.text+0x3b6a): undefined reference to `correct_yolo_boxes'
obj/parser.o: In function `parse_yolo':
parser.c:(.text+0x14da): undefined reference to `make_yolo_layer'
obj/parser.o: In function `parse_reorg_old':
parser.c:(.text+0x2958): undefined reference to `make_reorg_old_layer'
obj/parser.o: In function `parse_upsample':
parser.c:(.text+0x301f): undefined reference to `make_upsample_layer'
obj/parser.o: In function `parse_network_cfg_custom':
parser.c:(.text+0x60a4): undefined reference to `make_upsample_layer'
parser.c:(.text+0x65b6): undefined reference to `make_reorg_old_layer'
obj/demo.o: In function `demo':
demo.c:(.text+0x68b): undefined reference to `get_capture_video_stream'
demo.c:(.text+0xa7e): undefined reference to `send_mjpeg'
demo.c:(.text+0xd64): undefined reference to `get_capture_webcam'
collect2: error: ld returned 1 exit status
Makefile:82: recipe for target 'darknet' failed
make: *** [darknet] Error 1`

any suggestion?

**Comment by AlexeyAB** [comment]
Try to remove Darknet from PC, and download again.
